include/dem/find_cell_neighbors.h

```diff
@@ -77,15 +108,41 @@ class FindCellNeighbors
    * contact candidates
    *
    * @param triangulation Triangulation to access the information of the cells
-   * @param cells_total_neighbor_list An unordered_map (with size of the local cell
-   * number) of vectors (all adjacent cells of each local cell)
+   * @param cells_total_neighbor_list An unordered_map (with size of the local
+   * cell number) of vectors (all adjacent cells of each local cell)
    */
 
   void
   find_full_cell_neighbors(
     const parallel::distributed::Triangulation<dim> &triangulation,
     typename DEM::dem_data_structures<dim>::cells_total_neighbor_list
       &cells_total_neighbor_list);
+
+private:
+  /**
+   * Generate a periodic neighbor cells list of the cell.
```

Comment:
I think this could be a bit more verbose. Can you give an example of what that contains?

Reply:
Also @brief keyword

Reply:
Do you know why sometimes there's no @brief? I still have the doc in CLion, with or without